mtgjson5/providers/scryfall/monolith.py

```diff
@@ -8,10 +8,15 @@
 import re
 import sys
 import time
-from typing import Any, Dict, List, Optional, Set, Union
+from datetime import datetime
```

Comment:
I prefer to use `import X` when possible, with the exception of typing and singleton